### PR TITLE
Feat/resource contrainst

### PR DESCRIPTION
**Pull request by giangbui**
Put constraints CPU and Memory on job pod

### CODE REVIEW

File: handlers/handler.go [hunk 1]
```diff
@@ -161,7 +163,7 @@ func (handler *SQSHandler) HandleSQSMessage(m *mq.Message) error {
 	}
 
 	// remove completed jobs
-	RemoveCompletedJobs(jobNameList)
+	//RemoveCompletedJobs(jobNameList)
```
Comment (zflamig) [inline handlers/handler.go]: Is this intentional?

Comment (giangbui) [reply handlers/handler.go]: Fixed in 58015ab91c638aad9adb29fb6e60cbc947327845
Thanks!

**Comment by zflamig** [comment]
Are there docs on how this is actually deployed with the indexs3client somewhere? I was looking on Friday and couldn't find anything...

**Comment by giangbui** [comment]
I update the wiki doc here
https://github.com/uc-cdis/cdis-wiki/pull/68

**Pull request merged**